website_and_docs/content/documentation/webdriver/bidirectional/_index.zh-cn.md

```diff
@@ -1,26 +1,31 @@
 ---
-title: "双向功能"
-linkTitle: "双向"
+title: "双向协议"
+linkTitle: "双向协议"
 weight: 16
 aliases: [
 "/documentation/zh-cn/webdriver/bidi_apis/",
 "/zh-cn/documentation/webdriver/bidi_apis/"
 ]
 ---
 
-Selenium is working with browser vendors to create the
-[WebDriver BiDirectional Protocol](https://w3c.github.io/webdriver-bidi/)
-as a means to provide a stable, cross-browser API that uses the bidirectional
-functionality useful for both browser automation generally and testing specifically.
-Before now, users seeking this functionality have had to rely on
-with all of its frustrations and limitations.
+Selenium正在与浏览器供应商合作创建
+[WebDriver双向协议](https://w3c.github.io/webdriver-bidi/) ,
+作为一种提供稳定的跨浏览器API的方法,
+该API使用双向协议处理
+各种浏览器的通用自动化以及特定测试的需求.
+在此之前, 寻求此功能的用户
+必须忍受当前实现的全部问题和局限.
 
-The traditional webdriver model of strict request/response commands will be supplemented
-with the ability to stream events from the user agent to the controlling software via WebSockets,
-better matching the evented nature of the browser DOM.
 
-Because it's a bad idea to tie your tests to a specific version of a specific browser,
-the Selenium project recommends using WebDriver BiDi wherever possible.
-However, until the spec is complete there are many useful things that the CDP offers.
-To help keep your tests independent and portable, Selenium offers some useful helper classes.
-At the moment, these use the CDP, but when we shall be using WebDriver Bidi as soon as possible
+严格限制请求响应命令的传统webdriver模型, 
```

Comment:
it may be a mistake, but we always write it as WebDriver AFAIK!

Reply:
Got, just replaced all places with WebDriver in bidirectional